packages/stargate/src/queries/queryclient.ts

```diff
@@ -1,8 +1,45 @@
 /* eslint-disable no-dupe-class-members, @typescript-eslint/ban-types, @typescript-eslint/naming-convention */
-import { toHex } from "@cosmjs/encoding";
+import { ics23, verifyExistence, verifyNonExistence } from "@confio/ics23";
+import { fromAscii, toAscii, toHex } from "@cosmjs/encoding";
+import { firstEvent } from "@cosmjs/stream";
 import { Client as TendermintClient } from "@cosmjs/tendermint-rpc";
 import { arrayContentEquals, assert, isNonNullObject } from "@cosmjs/utils";
 
+// TODO: import these from @confio/ics23 when exported
```

Comment:
Do you create an update for this?

Reply:
Published https://www.npmjs.com/package/@confio/ics23/v/0.6.3